packages/ffe-icons/icons/hus-hammer-ikon.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 56 50" xmlns="http://www.w3.org/2000/svg">
```

Comment:
`viewBox` bør helst være `0 0 200 200`, eller i alle fall kvadratisk. For å fylle kvadraten kan det heller legges inn whitespace enten på sidene eller topp/bunn, alt ettersom. Dette gjør ikonet litt mer forutsigbart i kontekst av andre ikoner, som alle er 200 x 200. :)

Reply:
Jo, men dersom jeg tar denne inn i Inkscape, så flyttes clip-rule="evenodd" som nå ligger i en path, til en style-rule som gjør at svg'en ikke kompilerer. Kanskje jeg skal prøve med figma

Reply:
Jeg skullle til å foreslå og bare redigere den i VSCode eller lignende (det er tross alt bare XML) men om jeg åpner den i Chrome (https://raw.githubusercontent.com/SpareBank1/designsystem/29b07d5240ad64f75fdf43f4825c0d7234a45e13/packages/ffe-icons/icons/hus-hammer-ikon.svg) ser den feil ut så da er det vanskelig å verifisere at man ikke ødelegger noe.

Reply:
Høh.. spennende. Den ser feil ut på Chrome i Ubuntu men riktig ut på Chrome i Windows...

Reply:
Usikker på hva Inkscape genererer i utgangspunktet, men anbefaler å sende output fra alle editorer gjennom [SVGOMG](https://jakearchibald.github.io/svgomg/) i alle fall. Kan hende den skreller vekk eventuelle feil i markupen også.